tests/integration/test_validate_config.py

```diff
@@ -241,6 +309,6 @@ def test_schema_validation(tmpdir, raw_package_config, valid, expected_output):
             output = PackitAPI.validate_package_config(Path("test_dir"))
             assert expected_output in output
         else:
-            with pytest.raises(PackitConfigException) as exc:
+            with pytest.raises(Exception) as exc_info:
                 PackitAPI.validate_package_config(Path("test_dir"))
-                assert expected_output in exc
+            assert expected_output in str(exc_info.value)
```

Comment:
Has this always been wrong (`exc` instead of `exc.value`)?

Reply:
yes, I had to fix some failing tests because of it.